pkg/query/listener.go

```diff
@@ -113,6 +117,23 @@ func getCriterionFields(key, op, right antlr.TerminalNode) (leftOp string, opera
 	return
 }
 
+func criterionDateToUTC(c *Criterion) error {
```

Comment:
why is this needed?

Reply:
For this test - https://github.com/Peripli/service-manager/pull/287/commits/5f850b569d699e5e07660c1f3e29eaca775dff47#diff-a88508bea7d1a156ea3b6cd0ca291d40R401 
Where the query has offset from UTC but in the db we store in utc